src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,149 @@
+package parser;
+
+import Dictionary.Word;
+import command.*;
+import exception.*;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+/**
+ * Creates a Command object after extracting information needed.
+ */
+public class Parser {
+
+    /**
+     * Extracts the command specified in the user input and creates the respective command objects.
+     * @param input user input from command line
+     * @return respective Command class objects
+     */
+    public static Command parse(String input) {
+        try {
+            String[] taskInfo = input.split(" ", 2);
+            if (taskInfo[0].equals("exit")) {
+                return new ExitCommand();
+            } else if (taskInfo[0].equals("help")) {
+                // CREATE A HELP COMMAND TO SHOW THE AVAILABLE INSTRUCTION
+                return null;
+            } else if (taskInfo[0].equals("add")) {
+                if (taskInfo.length == 1) {
```

Comment:
Will be helpful to have a comment describing the string format you're looking for